src/app/workflow/workflow-file-editor/workflow-file-editor.component.ts

```diff
@@ -107,7 +107,14 @@ export class WorkflowFileEditorComponent extends FileEditing {
           this.workflowsService.getWorkflow(editedWorkflow.id).subscribe(
             (newlyGottenWorkflow: Workflow) => {
               this.workflowService.setWorkflow(newlyGottenWorkflow);
-              this.alertService.detailedSuccess();
+              const mostRecentId = newlyGottenWorkflow.workflowVersions.reduce(
```

Comment:
This `reduce` looks the same as the code on line 67. Could you create a common function?